ui/de-lib/src/main/java/org/iplantc/de/apps/client/models/AppCategoryStringValueProvider.java

```diff
@@ -11,7 +11,7 @@
 
     @Override
     public String getValue(AppCategory object) {
-        return object.getName() + " (" + object.getAppCount() + ")";
+        return object.getName();
```

Comment:
I thought I remembered us having a conversation about this.  Since we can't have counts for how many apps are in the hierarchies, we would get rid of the app counts entirely... something along those lines.